.github/CODEOWNERS

```diff
@@ -129,6 +129,8 @@ pkg/winperfcounters/                                     @open-telemetry/collect
 pkg/batchperresourceattr/                                @open-telemetry/collector-contrib-approvers @atoulme @dmitryax
 pkg/experimentalmetricmetadata/                          @open-telemetry/collector-contrib-approvers @rmfitzpatrick
 
+override/aws/                                            @open-telemetry/collector-contrib-approvers @sethAmazon
```

Comment:
should we have multiple code owners?

Reply:
The reality is we will never be able to upstream this change. We need a way to figure out how to remove this. This is here just so ccwia emf does not come out doe sev2 style.

I added this because it fails the checks without it.